src/styles/styles.css

```
@@ -0,0 +1,124 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.wrapper {
+  margin-right: auto;
+  margin-left: auto;
+  width: 60%;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: solid 1px #e8d2d2;
+  padding: 1.8rem 0;
+}
+
+.header__slogan {
+  color: #827b7b;
+  text-transform: uppercase;
+  font-weight: 100;
+  font-size: 1.7rem;
+  letter-spacing: 1px;
+  margin: 0 2rem;
+  text-align: center;
+}
+
+.filters {
+  display: grid;
+  grid-template-columns: 2fr minmax(50px, 1fr) minmax(50px, 1fr);
+  margin: 2rem auto 3rem;
+  grid-gap: 2rem;
+}
+
+.input {
+  border: solid 2px #e8d2d2;
+  border-radius: 5px;
+  height: 3rem;
+  font-size: 1.5rem;
+  padding: 0 0.8rem;
+  color: #827b7b;
+  grid-column: 1 / 2;
+}
+
+.filters-dropdown {
+  border: solid 2px #e8d2d2;
+  border-radius: 5px;
+  height: 3rem;
+  background: transparent;
+  font-size: 1.5rem;
+  padding: 0 0.8rem;
+  color: #827b7b;
+}
+
+.grid-col-4 {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(24rem, 1fr));
+  gap: 2.8rem;
+  align-items: center;
+  padding: 0;
+}
+
+.cards__list {
+  list-style: none;
+  border: solid 1px #f9f2f2;
+  border-radius: 10px;
+  display: flex;
+    flex-direction: column;
```

Corrigir indentação:

```suggestion
  flex-direction: column;
```